trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -46,6 +47,14 @@ const getPortalDeal = async (dealId, dealType) => {
  * Azure function
  */
 const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
+  /**
+   * 0.2s (200ms) delay
+   * Below is added to avoid any race condition being build up,
+   * where portal deal updates `deals` collection will not be
+   * fetched when updated on close proximity calls.
+   */
+  await delay(() => {}, 200);
```

Comment:
Lodash will delay execution of `() => {}`, but not main thread.

lodash.delay returns timerId, so execution of `() => {}` can be cancelled, but this is not promise. I don't think await will wait timer to execute.